aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/endpoints/OperationInputTestGenerator.kt

```diff
@@ -147,7 +148,7 @@ class OperationInputTestGenerator(_ctx: ClientCodegenContext, private val test:
                 #{assertion}
                 """,
                 "capture_request" to CargoDependency.smithyClient(runtimeConfig)
-                    .withFeature("test-util").toType().resolve("test_connection::capture_request"),
+                    .copy(features = setOf("test-util"), scope = DependencyScope.Dev).toType().resolve("test_connection::capture_request"),
```

Comment:
It took me a very long time to find the issue on this one. This is required otherwise it includes the feature to the production dependency and that breaks the integration test for wasm.

Reply:
OK. I think we should add a function to `RuntimeType` to centralize this logic then (there should be plenty of examples like this in there for you to reference).